lib/types/operation.js

```diff
@@ -242,6 +244,10 @@ Operation.prototype.getType = function (param) {
         return ref;
       }
     } else {
+      // If inline schema, we add it our interal hash -> which gives us it's ID (int)
+      if(schema.type === 'object') {
+        return 'inline#'+ this.addInlineModel(schema);
```

Comment:
I really think this should be modeled after how we handle inline schemas in responses, for consistency of course..  So instead of `inline#{num}` it would be `Inline Model {num}`.